src/components/customLayouts/RowLayout/RowLayoutRenderer.tsx

```diff
@@ -1,15 +1,15 @@
 import React from 'react';
-import { ResolvedJsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
+import { JsonFormsDispatch, withJsonFormsLayoutProps } from '@jsonforms/react';
 import { LayoutProps, Layout } from '@jsonforms/core';
 import RowStyles from './RowStyles';
 
 const RowLayoutRenderer = ({ schema, uischema, path, renderers, cells, enabled, visible }: LayoutProps) => (
-	<div>
+	<div data-testid='row-layout'>
 		{visible &&
 			(uischema as Layout).elements.map((child, idx) => {
 				return (
 					<RowStyles key={`row-${idx}`}>
-						<ResolvedJsonFormsDispatch
+						<JsonFormsDispatch
```

Comment:
Why this?

Reply:
It said that since Version 3.0 this optimization component is no longer necessary. Use JsonFormsDispatch instead.